### PR TITLE
[Tchap-v2] Update some display features

### PR DESCRIPTION
**Pull request by Yoshiin**
Update some display features.

* #89 
* #185 
* #118 
* #139 

See #201 for more informations.

### CODE REVIEW

File: src/stores/RoomListStore.js [hunk 1]
```diff
@@ -573,9 +576,11 @@ class RoomListStore extends Store {
                         if (LIST_ORDERS[tagName] === 'recent') category = this._calculateCategory(room);
                         lists[tagName].push({room, category: category});
                     }
-                } else if (dmRoomMap.getUserIdForRoomId(room.roomId)) {
+                } else if (dmUserId) {
                     // "Direct Message" rooms (that we're still in and that aren't otherwise tagged)
-                    lists["im.vector.fake.direct"].push({room, category: this._calculateCategory(room)});
+                    if (!Email.looksValid(dmUserId)) {
```
Comment (giomfo) [inline src/stores/RoomListStore.js]: Can you remove the existing comment "// "Direct Message"..."
and mention here that we decided to hide the direct room created to invite a contact by his email.
Note: The room should appear in the list when an account is created with this email

File: src/components/views/rooms/RoomPreviewBar.js [hunk 2]
```diff
@@ -74,16 +74,25 @@ module.exports = React.createClass({
...
+                }).done((result) => {
+                    this.setState({invitedEmailMxid: result.mxid});
+                }, (err) => {
+                    this.setState({threePidFetchError: err});
+                });
+            }
+            MatrixClientPeg.get().getProfileInfo(
+                this.props.inviterName
             ).finally(() => {
                 this.setState({busy: false});
```
Comment (giomfo) [inline src/components/views/rooms/RoomPreviewBar.js]: Note: `busy` is set to `false` at two different places. Is that normal?

Comment (Yoshiin) [reply src/components/views/rooms/RoomPreviewBar.js]: True

**Comment by Yoshiin** [comment]
PR closed due to some changes on files making conflict resolution too difficult.
New PR : #232

**Pull request closed without merging**